src/main/java/uk/co/ramp/covid/simulation/CovidParameters.java

```
@@ -14,7 +14,9 @@
     public static CovidParameters cparams = null;
 
     public static CovidParameters get() {
-        assert cparams != null : "Parameters have not yet been initialised";
+        if (cparams == null) {
```

I think this is a problem and we should throw an exception.